{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"other"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsauditmanager\/auditmanagerclienttypes\/validationexceptionreason\/other"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/AuditManagerClientTypes\/ValidationExceptionReason\/other","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"other"}],"title":"AuditManagerClientTypes.ValidationExceptionReason.other","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:15AWSAuditManager05AuditB11ClientTypesO25ValidationExceptionReasonO5otheryA2EmF","modules":[{"name":"AWSAuditManager"}]},"hierarchy":{"paths":[["doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager","doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/AuditManagerClientTypes","doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/AuditManagerClientTypes\/ValidationExceptionReason"]]},"references":{"doc://AWSAuditManager/documentation/AWSAuditManager/AuditManagerClientTypes/ValidationExceptionReason":{"role":"symbol","title":"AuditManagerClientTypes.ValidationExceptionReason","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ValidationExceptionReason"}],"abstract":[],"identifier":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/AuditManagerClientTypes\/ValidationExceptionReason","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ValidationExceptionReason"}],"url":"\/documentation\/awsauditmanager\/auditmanagerclienttypes\/validationexceptionreason"},"doc://AWSAuditManager/documentation/AWSAuditManager/AuditManagerClientTypes":{"role":"symbol","title":"AuditManagerClientTypes","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"AuditManagerClientTypes"}],"abstract":[],"identifier":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/AuditManagerClientTypes","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"AuditManagerClientTypes"}],"url":"\/documentation\/awsauditmanager\/auditmanagerclienttypes"},"doc://AWSAuditManager/documentation/AWSAuditManager":{"role":"collection","title":"AWSAuditManager","abstract":[],"identifier":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager","kind":"symbol","type":"topic","url":"\/documentation\/awsauditmanager"},"doc://AWSAuditManager/documentation/AWSAuditManager/AuditManagerClientTypes/ValidationExceptionReason/other":{"role":"symbol","title":"AuditManagerClientTypes.ValidationExceptionReason.other","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"other"}],"abstract":[],"identifier":"doc:\/\/AWSAuditManager\/documentation\/AWSAuditManager\/AuditManagerClientTypes\/ValidationExceptionReason\/other","kind":"symbol","type":"topic","url":"\/documentation\/awsauditmanager\/auditmanagerclienttypes\/validationexceptionreason\/other"}}}